WordPress/Classes/ViewRelated/Reader/Tab Navigation/ReaderTabViewModel.swift

```diff
@@ -91,6 +88,7 @@ extension ReaderTabViewModel {
 
         self.tabSelectionCallback = { [weak controller] topic in
             controller?.setTopic(topic)
+            controller?.isSavedPostsController = topic == nil ? true : false
```

Comment:
I don't think the tertiary is necessary. Couldn't it just be this?

`controller?.isSavedPostsController = (topic == nil)`

Reply:
👍